iguana.corecontroller/src/main/resources/iguana-schema.json

```diff
@@ -306,34 +224,24 @@
             }
           },
           "then": {
-            "allOf": [{
             "required": [
```

Comment:
I was unsure of the reason for the "allOf" in this one location (and not the others). This should have no effect since all the sub-components are included in any case.